fotoapparat/src/main/java/io/fotoapparat/parameter/factory/ParametersFactory.java

```diff
@@ -100,12 +107,42 @@ public static Parameters selectSensorSensitivity(@NonNull Capabilities capabilit
      * @param jpegQuality integer (1-100)
      * @return new parameters with a set jpegQuality
      */
-    public static Parameters selectJpegQuality(@NonNull Integer jpegQuality) {
+    public static Parameters selectJpegQuality(int jpegQuality) {
         return new Parameters().putValue(
                 Parameters.Type.JPEG_QUALITY,
-                jpegQuality
+                ensureJpegQualityRange(jpegQuality)
         );
     }
 
+    private static Integer ensureJpegQualityRange(int jpegQuality) {
+        if (jpegQuality < 1 || 100 < jpegQuality) {
```

Comment:
Should this be "< 0" instead of "< 1"? The default value for jpegQuality is zero so it's causing a fatal IllegalArgumentException crash if the user doesn't explicitly set a different jpegQuality value in their FotoapparatBuilder.